common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4144,6 +4144,29 @@
     ],
     "sqlState" : "42601"
   },
+  "STREAMING_OUTPUT_MODE" : {
```

Comment:
nit: Shall we start with `INVALID_`? I know you want to put this in subclass, but the general error message does not go with general error class.
@MaxGekk Do we happen to have similar case, and if we had how did we deal with it?

Reply:
> @MaxGekk Do we happen to have similar case, and if we had how did we deal with it?

We had, for instance:
https://github.com/apache/spark/blob/02795a36171cac51b1beaf68c42e8dfe3e6f0903/common/utils/src/main/resources/error/error-conditions.json#L653-L663

Reply:
@HeartSaVioR I believe it is not necessary to add the prefixes like `ERROR`, `INVALID`, `FAILED` everywhere especially in this particular case. It should be clear to users that we deal with error situation/condition, so, in our case the condition is `streaming + output mode`. And details like `INVALID`, `NOT_SUPPORTED` are in sub-classes.

Reply:
I don't have strong argument for this. What I am saying is the general message of the top-class is already denoting "invalid" output mode, so subclasses are expected to be a case of "invalid", not others.